examples/chatbot_llm/main.py

```diff
@@ -0,0 +1,25 @@
+import argparse
+from deepsparse import Pipeline
+
+def run(args):
+
+    pipeline = Pipeline.create(
+        task="text-generation",
+        model_path=args.model_path,
+        max_generated_tokens = args.max_generated_tokens,
+    )
+
+    while True:
+        # get input from user
+        input_text = input("User: ")
+        response = pipeline(sequences=[input_text], session_ids=args.session_id)
```

Comment:
What's the point of having session id as an arg?

Reply:
Yeah, for this small example probably should be a fixed string.